.github/workflows/dev-deploy-api.yml

```
@@ -45,9 +46,11 @@ jobs:
         name: ['novu/api-ee', 'novu/api']
     steps:
       - uses: actions/checkout@v3
+      - uses: ./.github/actions/checkout-submodules
         with:
-          submodules: ${{ contains (matrix.name,'ee') }}
-          token: ${{ secrets.SUBMODULES_TOKEN }}
+          run: ${{ contains (matrix.name,'ee') }}
```

```suggestion
          enabled: ${{ contains (matrix.name,'ee') }}
```

Another option, but current also make sense!

---

Actually i wanted to find another name because 'run' felt like a 'reserved word' for github actions.